platforms/software/build-init/src/main/java/org/gradle/buildinit/plugins/internal/modifiers/BuildInitTestFramework.java

```diff
@@ -27,7 +27,9 @@ public enum BuildInitTestFramework implements WithIdentifier {
     SPOCK("Spock"),
     KOTLINTEST("kotlin.test"),
     SCALATEST("ScalaTest"),
-    JUNIT_JUPITER("JUnit Jupiter");
+    JUNIT_JUPITER("JUnit Jupiter"),
+    XCTest("XCTest"),
```

Comment:
❓ Is it intentional that the enum value naming here is not SNAKE_CASE?

Reply:
No, it's not.

Reply:
Fixed